src/types/asgardex.d.tsx

```diff
@@ -11,3 +11,8 @@ export type Pair = {
   source: Maybe<string>
   target: Maybe<string>
 }
+
+export type AssetPair = {
+  asset: string
+  [key: string]: FixmeType
```

Comment:
[question] Don't we know the type for this ^ ? We should try to avoid to use `FixmeType`, which says what it is: FIXME ;)